Lib/test/test_email/test_email.py

```diff
@@ -3313,6 +3316,78 @@ def test_parsing_errors(self):
         # Test email.utils.supports_strict_parsing attribute
         self.assertEqual(email.utils.supports_strict_parsing, True)
 
+    def test_parsing_errors_strict_disabled_via_env_var(self):
+        address = 'alice@example.org )Alice('
+        empty = ('', '')
```

Comment:
For the test, you can just call _use_strict_email_parsing() rather than testing email parsing.

Reply:
You are right but I consider this more like an end-to-end test.